CHANGELOG.md

```diff
@@ -22,10 +22,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 - [Validation] Add a YAML file containing error information. [#117]
 - [Validation] Check whether a string is valid XML - truthy. [#45]
 - [Validation] Check whether a string is valid XML - detailed error information. [#45]
-- [Validation] Provide custom error messages for lxml errors stating that a string is not XML caused by published IATI data. [#90]
+- [Validation] Provide custom error messages when lxml returns errors for a string that is not XML. [#90]
 - [Validation] Check whether a Dataset is valid against an IATI Schema - truthy. [#45]
 - [Validation] Check whether a Dataset is valid against an IATI Schema - detailed error information. [#45]
-- [Validation] Provide custom error messages for lxml errors stating that a Dataset is not IATI XML caused by published IATI data. [#92]
+- [Validation] Provide custom error messages when lxml returns errors for Datasets that is do not contain valid IATI XML. [#92]
```

Comment:
"that is do not" -> `that do not`

Reply:
72ee53f0c7be8b9e45a3bfead1cf8106787dc87b